search/ranking_info.hpp

```diff
@@ -1,11 +1,10 @@
 #pragma once
 
+#include "search/common.hpp"
 #include "search/model.hpp"
 #include "search/pre_ranking_info.hpp"
 #include "search/ranking_utils.hpp"
 
-#include "std/iostream.hpp"
```

Comment:
`#include <iostream>` ?

Reply:
As you can see, it was removed.

Reply:
lines 40-42:
```C++
static void PrintCSVHeader(ostream & os);
void ToCSV(ostream & os) const;
```

Reply:
Looks like this file was touched occasionally :) Fixed anyway.